pyfar/dsp/filter/gammatone.py

```diff
@@ -161,8 +182,21 @@ def __eq__(self, other):
 
     @property
     def freq_range(self):
```

Comment:
here we need to add a new propertie called frequency_range and we keep to old one and throwing the warning.